ITensorGPU/test/runtests.jl

```diff
@@ -1,6 +1,22 @@
-using ITensorGPU, Test, CUDA
+if VERSION < v"1.8"
+  itensorgpu_path = joinpath(pwd(), "..")
```

Comment:
This makes a path relative to where the file is called from, so will probably fail if someone includes the `"runtests.jl"` file if they are in a different directory from `test/`, for example if they were in the `ITensorGPU/` directory and called `include("test/runtests.jl")`.

This should be more general:
```julia
itensorgpu_path = joinpath(@__FILE__, "..")
```
`@__FILE__` is a macro that expands to the path of the file where it is located, see [the Julia docs](https://docs.julialang.org/en/v1/base/base/#Base.@__FILE__) for more details.

Reply:
Ahh good point. I was only testing by launching `Pkg.test` and didn't consider `include`